bvm/ballerina-core/src/main/resources/MessagesBundle_en_US.properties

```diff
@@ -146,3 +146,4 @@ invalid.format.specifier = unknown format conversion ''{0}''
 invalid.task.config = Task scheduling configuration is invalid
 task.already.running = The task is already running
 task.not.running = The task in not running
+illegal.format.conversion = illegal format conversion ''{0}''
```

Comment:
shall we be consistent with "MessagesBundle.properties"? "**I**llegal" vs "**i**llegal"?

Reply:
Since we use %s to give string representation for all supported types in Ballerina, is it a good idea to make everything uppercase? I mean it would uppercase xml/json value if someone tries to print it using %S